pipelines/docker-build-dance/patch.yaml

```diff
@@ -23,7 +23,7 @@
 - op: replace
   path: /spec/tasks/3/taskRef
   value:
-    name: buildah
+    name: buildah-dance
```

Comment:
Should we stick everywhere to rhtap name?

Reply:
Fixed it.